app/src/main/res/values-ro/strings.xml

```diff
@@ -0,0 +1,105 @@
+<resources>
+    <string name="app_name" translatable="false">Myne</string>
+    <!-- Others Strings -->
+    <string name="not_applicable">N/A</string>
+    <string name="woops">"Ups!"</string>
+    <string name="dialog_confirm_button">Confirmă</string>
+    <string name="cancel">Anulează</string>
+    <string name="storage_perm_error">Te rugăm să acorzi permisiunea de stocare!</string>
+    <string name="no_app_to_handle_epub">Nu pot găsi nicio aplicație pentru a deschide un fișier epub.</string>
+    <string name="network_error">Ups! Nu a reușit să se conecteze cu serverul.</string>
+    <string name="network_error_retry">Încearcă din nou</string>
+    <string name="error">Ups! Ceva a mers rău.</string>
+
+    <!-- Navigation titles -->
+    <string name="navigation_home">Acasă</string>
+    <string name="navigation_categories">Categorii</string>
+    <string name="navigation_library">Librărie</string>
+    <string name="navigation_settings">Setări</string>
+
+    <!-- Welcome Screen -->
+    <string name="welcome_screen_text">Mulțumim că ai instalat Myne!\nTe rugăm să selectezi e-book reader-ul tău preferat pentru a trece la viteză.</string>
+    <string name="welcome_screen_button">Începeți</string>
+
+    <!-- Home screen. -->
+    <string name="home_header">Toate Cărțile</string>
+    <string name="home_language_icon_desc">Sortează după limbă</string>
+    <string name="home_search_icon_desc">Caută cărți</string>
+    <string name="language_menu_title">Răsfoiește după limbă</string>
+
+    <!-- Categories screen. -->
+    <string name="categories_header">Categorii</string>
+    <string name="no_books_found_for_lang_and_cat">Nu există cărți disponibile în limba %s pentru această categorie.</string>
+
+    <!-- Library screen. -->
+    <string name="library_header">Librărie</string>
+    <string name="empty_library">Nimic aici!</string>
+    <string name="open_app_chooser">Deschide cu…</string>
+    <string name="share_app_chooser">Împărtășiți cu…/string>
```

Comment:
Please close `</string>` tag properly at L38

Reply:
Fixed it now, can't believe that I didn't even notice it.

Reply:
Thanks!